src/main/assets/js/court-types.ts

```diff
@@ -64,34 +64,25 @@ export class CourtTypesController {
   }
 
   //added below methods to make sure local authorities tabs is enabled and disabled when family court type is updated.
-
-
   private getAreasOfLaw(): void{
     const slug = $('#slug').val();
     $.ajax({
       url: `/courts/${slug}/local-authorities-areas-of-law`,
       method: 'get',
       success: async (res) => {
         await this.updateContent(res, this.localAuthoritiesContentId);
-        this.disableLocalAuthoritiesTab();
+        Utilities.toggleTabEnabled(this.localAuthoritiesTabId, Boolean($('#enabled').val() === 'true'));
+        this.togglePostcodesTab();
       },
       error: (jqxhr, errorTextStatus, err) =>
         AjaxErrorHandler.handleError(jqxhr, 'GET local authorities areas of law failed.')
     });
-
   }
 
-  private disableLocalAuthoritiesTab(): void{
-    const isEnabled = Boolean($('#enabled').val() === 'true');
-    if(!isEnabled) {
-      $(this.localAuthoritiesTabId).addClass('disable-tab');
-      $(this.localAuthoritiesTabId).attr('disabled') ;
-    }
-    else
-    {
-      $(this.localAuthoritiesTabId).removeClass('disable-tab');
-      $(this.localAuthoritiesTabId).removeAttr('disabled') ;
-    }
-  }
+  // In the event that the court type is updated to county court, enable the postcodes tab if it has been disabled
+  private togglePostcodesTab(): void{
 
+    Utilities.toggleTabEnabled(this.postcodesNavTab,
+      Utilities.isCheckboxItemSelected('[name="types"]', 'County Court'));
```

Comment:
Are you using name because there are many checkboxes with the same name? If not, would using ID be better?

Reply:
ID comes back with a name like checkbox_1, which unfortunately doesn't match

Reply:
This will match any element in the dom with name of 'types'. If you need to use name, you could narrow it down using something like '#courtTypesContent input[name="types"]'

Reply:
I think its based on the way majid's code has been setup, id's for types has a prefix which kind of conflicts. I have tried adding an id of types in, and it doesn't work, open to suggestions!

Reply:
Changed :-)